tcf_website/management/commands/load_grades.py

```diff
@@ -37,78 +70,106 @@ def add_arguments(self, parser):
             type=str
         )
 
+        parser.add_argument(
+            '--suppress-tqdm',
+            action='store_true',
+            help='Suppress the tqdm loading bars'
+        )
+
+        parser.add_argument(
+            '--log-missing-instructors',
+            action='store_true',
+            help='Create a .txt file with missing instructors'
+        )
+
     def handle(self, *args, **options):
+        """Standard Django function implementation - runs when this command is executed."""
         self.verbosity = options['verbosity']
-        self.data_dir = 'tcf_website/management/commands/grade_data/csv/'
+        self.suppress_tqdm = options['suppress_tqdm']
+        self.log_missing_instructors = options['log_missing_instructors']
+
         if self.verbosity > 0:
             print('Step 1: Fetch Course and Instructor data for later use')
-        self.courses = {
-            (obj['subdepartment__mnemonic'], obj['number']): obj['id']
-            for obj in Course.objects.values('id', 'number',
-                                             'subdepartment__mnemonic')
-        }
-        self.instructors = {
-            (obj['first_name'], obj['last_name'], obj['email']): obj['id']
-            for obj in Instructor.objects.values('id', 'first_name',
-                                                 'last_name', 'email')
-        }
         semester = options['semester']
         if semester == 'ALL_DANGEROUS':
-            # ignores temporary files ('~' on Windows, '.' otherwise)
+            # ALL_DANGEROUS removes all existing data
+            CourseGrade.objects.all().delete()
+            CourseInstructorGrade.objects.all().delete()
+
+            # Loads every data CSV file in /grade_data/csv with exceptions
             for file in sorted(os.listdir(self.data_dir)):
-                if file[0] not in ('.', '~'):
+                if self.verbosity == 3:
+                    print('Loading data from', file)
+                # Ignore temp files (start with '~' on Windows, '.' otherwise) and test data
+                if file[0] not in ('.', '~') and file != 'test_data.csv':
```

Comment:
In either case, I think you'd have to hardcode _something_ related to the test file's path, since `manage.py load_grades {FILENAME}` expects `FILENAME` to be located in the `csv` folder, so if we made a separate directory the test command would be something like `manage.py load_grades ../test/test_data.csv` which also doesn't feel great. Arguably that refactors the jank into test code that isn't looked at as often...but I don't think that's a great approach either.